src/index.js

```diff
@@ -5,8 +5,6 @@ import 'leaflet/dist/leaflet.css';
 import 'leaflet.locatecontrol/dist/L.Control.Locate.css';
 import './bootstrap.less';
 
-export colors from './colors';
```

Comment:
Why did you remove that export?

Reply:
It is now exported within `src/constants/index.js`.
I thought it was more consistant to have all constants in a dedicated folder.

Reply:
Oh you're right, my bad! ^^